src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -188,6 +188,10 @@ var DOMPropertyOperations = {
     }
     if (value == null) {
       node.removeAttribute(name);
+    } else if (value === true) {
+      node.setAttribute(name, '');
+    } else if (value === false) {
+      node.removeAttribute(name);
```

Comment:
Are there any other enumerated attributes like that which use stringified booleans? I would hate to special case them, but I also think it's reasonable to ask users to provide the expected stringified values themselves, e.g., `spellcheck="true"`. We could have dev-only warnings for that maybe.